src/cuda/CMakeLists.txt

```diff
@@ -22,8 +22,7 @@ set(CUFINUFFT_INCLUDE_DIRS
     ${PROJECT_SOURCE_DIR}/include
     ${PROJECT_SOURCE_DIR}/contrib
     $<TARGET_PROPERTY:CUDA::cudart,INTERFACE_INCLUDE_DIRECTORIES>
-    $<TARGET_PROPERTY:CUDA::cufft,INTERFACE_INCLUDE_DIRECTORIES>
-    $<TARGET_PROPERTY:CUDA::nvToolsExt,INTERFACE_INCLUDE_DIRECTORIES>)
+    $<TARGET_PROPERTY:CUDA::cufft,INTERFACE_INCLUDE_DIRECTORIES>)
```

Comment:
Would this deletion cause compilation issues on older cuda versions?

Reply:
I am not sure about which header from `CUDA::nvToolsExt` is included, do you have CUDA 11 machine to test it?